premailer/premailer.py

```diff
@@ -114,7 +114,8 @@ def __init__(self, html, base_url=None,
                  cache_css_parsing=True,
                  cssutils_logging_handler=None,
                  cssutils_logging_level=None,
-                 disable_leftover_css=False):
+                 disable_leftover_css=False,
+                 align_floating_images=False):
```

Comment:
I'm not sure this should be off by default.
It seems like such a useful feature. If it's off by default only people with deep insight into Outlook will benefit and others will miss out. 
Let's just change it to by True by default.